arnold-project/create.sh

```diff
@@ -0,0 +1,26 @@
+echo "Creating first template"
+npx create-react-app arnold-project
+
+cd arnold-project
+cd src
+mkdir assets
```

Comment:
Could you please use mkdir -p to create subfolders.

Reply:
I've made changes to create.sh using mkdir -p to create all subfolders.